lib/player.rb

```diff
@@ -0,0 +1,19 @@
+# inputs : player name(string) ** outputs:
+class Player
+  POSSIBILITIES = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]].freeze
+  attr_accessor :name, :picks_array, :totem
```

Comment:
- [ ] use attr_accessor only where's needed. I can see that the player class has is using attr_accessor for the name, but it's only set up on the constructor. Instead, use attr_reader.